server/controllers/list.js

```diff
@@ -157,27 +143,16 @@ const addItemToList = (req, resp) => {
             'An error occurred while adding a new item. Please try again.'
         });
       }
-
+      const listInstance = new List();
```

Comment:
I think this was a bad idea to create instance's method (**I know that was my idea an my fault**) because in this case we don't operate on specific list's document (you have to create new one to use it)... Code after change looks strange. Maybe it would be better to move this 'helper' functions to utilities....